internal/handlers/telegram/helpers.go

```diff
@@ -5,12 +5,25 @@ import (
 )
 
 /*
-GetUsername returns the name and username of a user. Since usernames are optional
+ResolveUserName does basic cleanup if a user does not have a username on Telegram.
+*/
+func ResolveUserName(u *tgbotapi.User) string {
+	if u.UserName == "" {
+		return u.FirstName
+	}
+	// Add ZWSP to prevent pinging across platforms
+	// See https://github.com/42wim/matterbridge/issues/175
+	username := u.UserName[:1] + "" + u.UserName[1:]
+	return username
+}
+
+/*
+GetFullUsername returns the name and username of a user. Since usernames are optional
 on Telegram, we first need to check to see if they have one set.
 */
-func GetUsername(u *tgbotapi.User) string {
+func GetFullUsername(u *tgbotapi.User) string {
 	if u.UserName == "" {
 		return u.FirstName
 	}
-	return u.FirstName + " (@" + u.UserName + ")"
+	return u.FirstName + " (@" + u.UserName[:1] + "" + u.UserName[1:] + ")"
```

Comment:
Dumb question but I want to be explicit. The `(@username)` part is only returned if a user has a username set, otherwise it is just their first name. Right?

Reply:
yep, since usernames are not required by Telegram, this will only return the First name (which is required) if no username is set.